src/Tickets/Admin/Tickets/List_Table.php

```diff
@@ -389,7 +419,7 @@ public function column_name( $item ): string {
 			return esc_html( $item->name );
 		}
 
-		$edit_post_url  = get_edit_post_link( $event );
+		$edit_post_url  = $this->get_event_edit_url( $event->ID );
 		$edit_post_link = sprintf(
 			'<a href="%s" class="tec-tickets-admin-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
 			esc_url( $edit_post_url ),
```

Comment:
Any reason to not jsut do:
```suggestion
		$edit_post_link = sprintf(
			'<a href="%s" class="tec-tickets-admin-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
			esc_url( $this->get_event_edit_url( $event->ID ) ),
```